x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/monitor_type_badge.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBadge, EuiIcon } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+import {
+  EncryptedSyntheticsMonitor,
+  ConfigKey,
+  FormMonitorType,
+  DataStream,
+} from '../../../../../../common/runtime_types';
+
+export function MonitorTypeBadge({ monitor }: { monitor: EncryptedSyntheticsMonitor }) {
+  return (
+    <EuiBadgeStyled>
+      <EuiIcon size="s" type={getMonitorTypeBadgeIcon(monitor)} />{' '}
+      {getMonitorTypeBadgeTitle(monitor)}
```

Comment:
would be cool to add `onClick` and filter list by that type.

Reply:
Sounds good. PR now includes this functionality.